brainscore/benchmarks/zhu2019.py

```diff
@@ -0,0 +1,211 @@
+import numpy as np
+from numpy.random import RandomState
+import brainscore
+from brainio.assemblies import DataAssembly
+from brainscore.benchmarks import BenchmarkBase
+from brainscore.benchmarks.screen import place_on_screen
+from brainscore.metrics import Score
+from brainscore.metrics.accuracy import Accuracy
+from brainscore.model_interface import BrainModel
+from brainscore.utils import LazyLoad
+from scipy.stats import pearsonr
+import csv
+from pathlib import Path
+
+BIBTEX = """@article{zhu2019robustness,
+            title={Robustness of object recognition under extreme occlusion in humans and computational models},
+            author={Zhu, Hongru and Tang, Peng and Park, Jeongho and Park, Soojin and Yuille, Alan},
+            journal={arXiv preprint arXiv:1905.04598},
+            year={2019}
+        }"""
+
+DATASETS = ['extreme_occlusion']
+NUM_SPLITS = 50
+
+
+class _Zhu2019Accuracy(BenchmarkBase):
+
+    # Behavioral benchmark: compares model to average humans.
+    # human ceiling is calculated by taking NUM_SPLIT split-half reliabilities with *category-level* responses.
+
+    def __init__(self):
+        self._assembly = LazyLoad(lambda: load_assembly('extreme_occlusion'))
+        self._fitting_stimuli = brainscore.get_stimulus_set('Zhu2019_extreme_occlusion')
+        self._stimulus_set = LazyLoad(lambda: load_assembly('extreme_occlusion').stimulus_set)
+        self._visual_degrees = 8
+        self._number_of_trials = 1
+        self._metric = Accuracy()
```

Comment:
```suggestion
        self._metric = AccuracyMatch()
```

Reply:
right now you're just computing the accuracy as far as I can tell, but we want the **match** to human accuracy